src/studio/src/designer/frontend/packages/schema-model/test/testUtils.ts

```diff
@@ -7,18 +7,7 @@ import { FieldType, Keywords } from '../src';
 /**
  * Some schemas might not be valid
  */
-export const ignoreTestSchemas: string[] = ['ComplexSchema-old', 'ComplexSchema'];
-
-// These filkes are ignored because failed
-ignoreTestSchemas.push(
-  'ComplexSchema',
-  'NestedArrays',
-  'NestedWithArraySequence',
-  'NillableAttribute',
-  'SeresArray',
-  'SeresNillable',
-  'SeresSimpleTypeRestrictions',
-);
+export const ignoreTestSchemas: string[] = ['ComplexSchema-old'];
```

Comment:
As an extension to previous comment,  deleting "ComplexSchema-old.json" file would avoid dealing with "specal case schemas".